### DISCUSSION TITLE
enable camera Image injection browserstack

### DISCUSSION THREAD

**Discussion by DoreyKiss** [webdriverio/webdriverio#12713] [Q&A] [answered]
'bstack:options': { enableCameraImageInjection: true,

is not in the wdio/browserstack-service as a capability.
Can i use the browserstack-node-sdk with wdio or is it planned to add this to the service?

**Comment by christian-bromann** [comment, marked as answer]
> is not in the wdio/browserstack-service as a capability.

What do you mean by that? You can pass in any capabilities you like. If you are using TypeScript and it shows an error , it is because the BS options don't have this type yet, feel free to send a pull request adding it to https://github.com/webdriverio/webdriverio/blob/146668cd7841179e4e0fea70e38d9df328f00379/packages/wdio-types/src/Capabilities.ts#L1444

**Comment by m-manoj-kumar** [comment]
Try adding capability as below:
```
capabilities: [{
    'bstack:options': {
      deviceName: 'Samsung Galaxy S22 Ultra',
      platformVersion: '12.0',
      platformName: 'android',
      enableCameraImageInjection: true,
    }
  }
```
  
  Here is the document for BrowserStack WDIO SDK [](https://www.browserstack.com/docs/app-automate/appium/getting-started/nodejs/webdriverio)

**Reply by DoreyKiss** [reply]
As mentioned before you, it is not in the list of capabilitites, so has to be added.
I can see the sdk but its not working since the option is not in the capabilities.ts file.
